Framework/MDAlgorithms/src/IntegrateEllipsoids.cpp

```diff
@@ -278,6 +278,16 @@ void IntegrateEllipsoids::init() {
       "IntegrateIfOnEdge", true,
       "Set to false to not integrate if peak radius is off edge of detector."
       "Background will be scaled if background radius is off edge.");
+
+  declareProperty("AdaptiveQBackground", false,
+                  "Default is false.   If true, "
+                  "BackgroundOuterRadius + AdaptiveQMultiplier * **|Q|**"
+                  "BackgroundInnerRadius + AdaptiveQMultiplier * **|Q|**");
+
+  declareProperty("AdaptiveQMultiplier", 0.0,
```

Comment:
I recently had to change this so instrument scientist could use negative values.  The intensity is set to zero for negative radius. #17945